internal/extender/unschedulablepods.go

```diff
@@ -55,13 +55,20 @@ func NewUnschedulablePodMarker(
 	podLister corelisters.PodLister,
 	coreClient corev1.CoreV1Interface,
 	overheadComputer *OverheadComputer,
-	binpacker *Binpacker) *UnschedulablePodMarker {
+	binpacker *Binpacker,
+	timeoutDuration time.Duration) *UnschedulablePodMarker {
+
+	if timeoutDuration == 0 {
```

Comment:
Should we check for negative values too?

Reply:
This is a check just for the default timeout value, but I think that makes sense.